File: demoflavius/demo_flavius_Core/Models/Message.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace demo_flavius_Core.Models
{
    public class Message
    {
        [JsonProperty(PropertyName = "id")]
        public string Id { get; set; }

        [JsonProperty(PropertyName = "content")]
        public string Content { get; set; }

        [JsonProperty(PropertyName = "fromId")]
        public string FromId { get; set; }

        [JsonProperty(PropertyName = "fromName")]
        public string FromName { get; set; }

        [JsonProperty(PropertyName = "fromPicture")]
        public string FromPicture { get; set; }

        [JsonProperty(PropertyName = "toId")]
        public string ToId { get; set; }

        [JsonProperty(PropertyName = "deviceType")]
        public Enums.DeviceType DeviceType { get; set; }
    }
}

File: demoflavius/Android Solution/demoflavius/src/com/example/demoflavius/data/Message.java
package com.example.demoflavius.data;

public class Message 
{
	String id;
	String content;
	String fromId;
	String fromName;
	String fromPicture;
	String toId;
	Enums.DeviceType deviceType;
	
	public String getId() {
		return id;
	}
	public void setId(String id) {
		this.id = id;
	}
	public String getContent() {
		return content;
	}
	public void setContent(String content) {
		this.content = content;
	}
	public String getFromId() {
		return fromId;
	}
	public void setFromId(String fromId) {
		this.fromId = fromId;
	}
	public String getFromName() {
		return fromName;
	}
	public void setFromName(String fromName) {
		this.fromName = fromName;
	}
	public String getToId() {
		return toId;
	}
	public void setToId(String toId) {
		this.toId = toId;
	}
	public Enums.DeviceType getDeviceType() {
		return deviceType;
	}
	public void setDeviceType(Enums.DeviceType deviceType) {
		this.deviceType = deviceType;
	}
	public String getFromPicture() {
		return fromPicture;
	}
	public void setFromPicture(String fromPicture) {
		this.fromPicture = fromPicture;
	}
}
